packages/core/redux/assetsHolder/actions.js

```diff
@@ -168,3 +170,45 @@ export const withdrawAsset = (amount: Amount, token: TokenModel, feeMultiplier:
     console.error('withdraw error', e)
   }
 }
+
+export const requireTIME = () => async (dispatch, getState) => {
+  try {
+    const state = getState()
+    const assetDonatorDAO = daoByType('AssetDonator')(state)
+
+    const tx = await assetDonatorDAO.requireTIME()
```

Comment:
The variables `state` and `assetDonatorDAO`. Each variable is used just once. The lines 176-179 can be written much shorter e.g.
```
const tx = await daoByType('AssetDonator')(getState()).requireTIME()
```

Reply:
Fixed